tests/test_apikey.py

```diff
@@ -9,6 +9,6 @@ def test_get_apikey(autoextract_env_variable):
     assert get_apikey() == autoextract_env_variable
 
 
-def test_get_apikey_missing():
+def test_get_apikey_missing(autoextract_blank_env_variable):
```

Comment:
hm, technically this is a different test case, and previous case (an absent env varibale) is not covered now